src/cartridges/int_adyen_overlay/cartridge/scripts/util/adyenHelper.js

```diff
@@ -204,6 +206,11 @@ var adyenHelperObj = {
     return availablePaymentMethods.indexOf(paymentMethod) !== -1;
   },
 
+  getBlockedPaymentMethods: function getBlockedPaymentMethods() {
```

Comment:
```suggestion
function getBlockedPaymentMethods() {
```

Reply:
that was the old way of declaring functions in SFCC, we're not following it anymore